src/Sarif.Sarifer.UnitTests/SpamBackgroundAnalyzerTests.cs

```diff
@@ -0,0 +1,60 @@
+// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Text;
+
+using FluentAssertions;
+
+using Microsoft.CodeAnalysis.Sarif.Sarifer;
+
+using Moq;
+
+using Xunit;
+
+namespace Sarif.Sarifer.UnitTests
+{
+    public class SpamBackgroundAnalyzerTests
+    {
+        [Fact]
+        public void LoadPatternFiles_WhenDirectoryDoesNotExist_ShouldReturnEmptyList()
+        {
+            var mock = new Mock<IFileSystem>();
+            mock.Setup(fs => fs.DirectoryExists(It.IsAny<string>())).Returns(() => false);
```

Comment:
>() => false [](start = 77, length = 11)

This is constant; doesn't need to be a lambda/closure, so just `Returns(false)`.